OrbitGl/DataView.h

```diff
@@ -33,7 +33,8 @@ class DataView {
   virtual bool IsSortingAllowed() { return true; }
   virtual const std::vector<SortingOrder>& GetColumnInitialOrders();
   virtual int GetDefaultSortingColumn() { return 0; }
-  virtual std::vector<std::wstring> GetContextMenu(int a_Index);
+  virtual std::vector<std::wstring> GetContextMenu(
+      int /*a_ClickedIndex*/, const std::vector<int>& /*a_SelectedIndices*/);
```

Comment:
you only need to comment parameters in case they are unused, like in L39.
Since this is a declaration - please keep them uncommented.

Reply:
Makes sense, done.